packages/react-components/react-breadcrumb/docs/Spec.md

```diff
@@ -2,54 +2,555 @@
+  <Breadcrumb aria-label="breadcrumb">
+    <BreadcrumbItem>
+      <BreadcrumbButton onClick={() => {}}>
+        Home
+      </BreadcrumbButton>
+    </BreadcrumbItem>
+    <BreadcrumbItem>
+      <BreadcrumbButton onClick={() => {}}>
+        Gallery
+      </BreadcrumbButton>
+    </BreadcrumbItem>
+    <BreadcrumbItem>
+      <BreadcrumbButton onClick={() => {}}>
+        About
+      </BreadcrumbButton>
+    </BreadcrumbItem>
+  </Breadcrumb>
+  </>
+);
+```
 
 ## Variants
 
-_Describe visual or functional variants of this control, if applicable. For example, a slider could have a 2D variant._
+### Appearance
 
-## API
+Breadcrumb can be:
+
+- transparent (default)
+- subtle
+
+### BreadcrumbItem variants
+
+- Non-clickable element which is BreadcrumbItem component.
+- Button - BreadcrumbButton component.
+- Link - BreadcrumbLink component.
+- Dropdown menu - can be added by a partner using JSX composition.
+
+### Icon
 
-_List the **Props** and **Slots** proposed for the component. Ideally this would just be a link to the component's `.types.ts` file_
+BreadcrumbItem can have an icon with text or just icon.
+
+### Size
+
+Breadcrumb can have the following sizes: `small`, `medium` and `large`.
+
+### Tooltip
+
+Tooltips appear on collapsed menu or truncated name of item.
+
+### Dropdown menu
+
+Dropdown contains collapsed items.
+
+## API
 
 ## Structure
 
-- _**Public**_
-- _**Internal**_
-- _**DOM** - how the component will be rendered as HTML elements_
+### Components
+
+| Component         | Purpose                                                                  |
+| ----------------- | ------------------------------------------------------------------------ |
+| Breadcrumb        | Wrapper for the Breadcrumb component. Contains `nav` and `ol` elements.  |
+| BreadcrumbDivider | Divider component                                                        |
+| BreadcrumbItem    | `li` element. Can contain BreadcrumbButton or BreadcrumbLink components. |
+| BreadcrumbLink    | Breadcrumb Link                                                          |
+| BreadcrumbButton  | Breadcrumb Button                                                        |
+
+### Breadcrumb
+
+#### Anatomy
+
+![visual anatomy of the Breadcrumb component](./assets/breadcrumb-anatomy.png)
+
+#### DOM
+
+```HTML
+<nav aria-label="breadcrumb">
+  <ol>
+  {children}
+  </ol>
+</nav>
+```
+
+#### Breadcrumb structure:
+
+- root `nav` element
+- list `ol` element
+
+#### API
+
+| Property   | Values                                  | Default       | Purpose                                     |
+| ---------- | --------------------------------------- | ------------- | ------------------------------------------- |
+| appearance | `transparent`, `subtle`                 | `transparent` | Sets appearance                             |
+| focusMode  | `off`, `no-tab`, `tab-exit`, `tab-only` | `off`         | Sets the focus behavior for the Breadcrumb. |
```

Comment:
```suggestion
| focusMode  | `none`, `no-tab`, `tab-exit`, `tab-only` | `none`         | Sets the focus behavior for the Breadcrumb. |
```

Let's be consistent with table and use `none`

Reply:
I see that there are different names in other components. For example, in Card, it's `off`. I'll, also, ask designers if we need an opportunity to turn it off. By default, it should be focusable. I'll update this part

Reply:
No need to add a focus configuration. It's turned-on by default